lib/src/shared/main/java/com/couchbase/lite/internal/core/C4Document.java

```diff
@@ -88,6 +88,8 @@ public FLDict getSelectedBody2() {
 
     // - Lifecycle
 
+    public int getSelectedFlags() { return withHandle(C4Document::getSelectedFlags, 0); }
```

Comment:
assuming this is selected _revision_ flag, and not document flag.

Reply:
Yeah... I followed it into the JNI.  It is the selected flag.